website/versioned_docs/version-0.4.x/guides/composedb-server/access-mainnet.mdx

```diff
@@ -0,0 +1,145 @@
+A valid email address is required so that you have a way to control the Ceramic nodes that are given access to the 3Box Labs anchor service. Using this email you will be able to register or revoke DIDs for your nodes.
+
+```bash
+> curl --request POST \
+  --url https://cas.3boxlabs.com/api/v0/auth/verification \
+  --header 'Content-Type: application/json' \
+  --data '{"email": "youremailaddress"}'
+```
+
+You should see a response that says `"Please check your email for your verification code."`
+
+Now check your email and copy the one time passcode enclosed within. It will be a string of letters and numbers _similar_ to this: **`2451cc10-5a39-494d-b8eb-1971ecd813de`.**
+
+## Step 3. Register your DID
+
+Use your DID, the one time passcode, and the same email address, to register your DID.
+
+```bash
+> curl --request POST \
+  --url https://cas.3boxlabs.com/api/v0/auth/did \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "email": "youremailaddress",
+	  "otp": "youronetimepasscode",
+	  "dids": [
+		  "yourdid"
+	  ]
+  }'
+```
+
+You should see a response that says `[{"email":"youremailaddress","did":"yourdid","nonce":"0","status":"Active"}]`.
+
+Finally, start your Ceramic node again. You will know that the DID registration was successful if you see logs in the console like the ones below.
+
+```bash
+IMPORTANT: Connected to anchor service 'https://cas.3boxlabs.com' with supported anchor chains ['eip155:1']
+IMPORTANT: Ceramic API running on 0.0.0.0:7007'
+```
+
+## Existing Node Operators
+
+Already running a node? Learn how to upgrade.
+    
+If you are already running a Ceramic daemon connected to mainnet, you have been using IP address based authentication to connect to the 3Box Labs mainnet CAS. You are not required to re-register or make any changes to your daemon, however please note that we will be deprecating IP address based authentication in the future. To prepare for deprecation, we recommend updating your daemon config file to use DID based authentication, then registering your DID with the steps above.
+
+If you have run a Ceramic daemon before but have not yet connected to mainnet, you must update your daemon config file to use DID based authentication, then register your DID with the steps above.
+
+### Updating to DID based authentication
+    
+First you will need to generate a private seed. You can do this with the ComposeDB CLI.
+    
+```bash
+> composedb did:generate-private-key
+✔ Generating random private key... Done!
+99918d7f36991ec38d76e1cf21d14c5348d1513512c957d0b809efbf3ad18983
+```
+
+Copy the string of numbers and letters logged to the console.
+
+:::note
+
+As a security best practice, do not use any private key or seed more than once.
+
+:::
+
+Update your daemon.config.json file to set your anchor auth method and node private seed url.
+
+```bash
+  "anchor": {
+    "auth-method": "did"
+  },
+```
+
+```bash
+  "node": {
+    "privateSeedUrl": "inplace:ed25519#privateseedcopiedfromabove"
```

Comment:
maybe change this `privateseedcopiedfromabove` to `private_seed_copied_from_above` so it reads better and will have the side effect of not being a valid seed so an error will be thrown if its not changed.